.github/workflows/test.yaml

```diff
@@ -58,11 +58,11 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
-      - name: Symlink lightwalletd and zcash binaries for darkside-tests
-        run: ln -s /usr/bin/lightwalletd ./darkside-tests/lightwalletd_bin/
+      - name: create binaries dir
+        run: mkdir -p ./zingo-testutils/test_binaries/bins
 
-      - name: Symlink lightwalletd and zcash binaries for integration-tests
-        run: ln -s /usr/bin/lightwalletd /usr/bin/zcashd /usr/bin/zcash-cli ./integration-tests/regtest/bin/
+      - name: Symlink lightwalletd and zcash binaries
+        run: ln -s /usr/bin/lightwalletd /usr/bin/zcashd /usr/bin/zcash-cli ./zingo-testutils/test_binaries/bins/
```

Comment:
should this create new symlinks automatically? tests would not run unless i manually symlinked lightwalletd, zcashd and  zcash-cli to zingo-testutils/test_binaries/bins/

Reply:
do you have git-lfs?

Reply:
yes i git reset --hard and it is downloading them them now so not sure what was up there.

Reply:
im getting an error running tests with them though. its trying to use glibc which i didn't have installed and isn't running even after installing glibc-source. could this be a problem with how the binaries were compiled or do i need to configure the system to use glibc? heres the error im getting for each test using zcashd:
 ``` 
FAIL [   0.508s] integration-tests::integrations slow::t_incoming_t_outgoing_disallowed

--- STDOUT:              integration-tests::integrations slow::t_incoming_t_outgoing_disallowed ---

running 1 test
test slow::t_incoming_t_outgoing_disallowed ... FAILED

failures:

failures:
    slow::t_incoming_t_outgoing_disallowed

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 52 filtered out; finished in 0.50s


--- STDERR:              integration-tests::integrations slow::t_incoming_t_outgoing_disallowed ---
[zingo-testutils/src/regtest.rs:324] "{:?}" = "{:?}"
[zingo-testutils/src/regtest.rs:324] &command.get_current_dir() = None
thread 'slow::t_incoming_t_outgoing_disallowed' panicked at /home/idky137/src/zingolib/zingo-testutils/src/lib.rs:319:33:
exit status: 1  /home/idky137/src/zingolib/zingo-testutils/test_binaries/bins/zcashd: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.38' not found (required by /home/idky137/src/zingolib/zingo-testutils/test_binaries/bins/zcashd)

note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
tests are running on my machine on dev so think it must be something to do with using the downloaded binary instead of one i compiled